engine/src/flutter/shell/platform/darwin/ios/framework/Headers/FlutterAppDelegate.h

```diff
@@ -24,8 +24,10 @@
  * code as necessary from FlutterAppDelegate.mm.
  */
 FLUTTER_DARWIN_EXPORT
-@interface FlutterAppDelegate
-    : UIResponder <UIApplicationDelegate, FlutterPluginRegistry, FlutterAppLifeCycleProvider>
+@interface FlutterAppDelegate : UIResponder <UIApplicationDelegate,
+                                             UIWindowSceneDelegate,
```

Comment:
I moved this into the public API because there are cases where users will have to override our implementation of `application:configurationForConnectingSceneSession:options:`.  I was hoping users could just modify the behavior by editing the info.plist but doesn't seem like it.

Reply:
The fact that we moved this into public API means that changing it again will be another breaking change. I'd suggest either 

(1) keep it private, or
(2) create a public `FlutterSceneDelegate` class and hook it up from `FlutterAppDelegate`.

Reply:
I don't think keeping it private is an option, there are cases where people will want to override the behavior.  A breaking change is defined as breaking one of our tests; I don't think that will be the case here.

Reply:
I'm having a hard time imagining a scenario where this would break a user.  Do you have one in mind?

Reply:
Removing it however will be a breaking change if we decide to refactor.  I'll poke around at the FlutterSceneDelegate option and report back.

Reply:
Wait, `application:configurationForConnectingSceneSession:options:` is on UIApplicationDelegate.  I got mixed up thinking it was on UISceneDelegate.  We can make the scene delegate private.  Done.